recipe/meta.yaml

```diff
@@ -6,8 +6,8 @@ package:
 
 source:
   fn: notebook-{{ version }}.tar.gz
-  url: https://github.com/jupyter/notebook/archive/{{ version }}.tar.gz
-  sha256: edb7ef8153e5ca781f21c7feb50b3316d515b0d8861ba0e45a00280166214a7f
+  url: https://pypi.python.org/packages/source/n/notebook/notebook-{{ version }}.tar.gz
+  md5: 3a72bb93f6f01caec69e5e65718f5f7f
```

Comment:
> but since the js packages aren't being extracted and packaged themselves as conda dependencies

Not yet, mwaha mwaha mwaha ha (evil laugh) :smiling_imp: 

I do lean towards the Debian approach on this, but am pragmatic about the cost of doing so (and gave up on doing it for ipywidgets as I couldn't get it to work on Windows).